modules/system/rolls/cast-test.js

```diff
@@ -141,8 +177,13 @@ export default class CastTest extends TestWFRP {
         }
       }
       //@/HOUSE
-
     }
+    //@HOUSE
+    if (this.preData.quickcasting && miscastCounter > 0) { 
+      this.result.other.push(game.i18n.localize("CHAT.Quickcasting"))
+      miscastCounter++;
+    }
+    //@/HOUSE
```

Comment:
Can you add  `game.wfrp4e.utility.logHomebrew("quickcasting")`

Reply:
done